core/src/Xios.cpp

```diff
@@ -44,10 +50,14 @@
 
 namespace Nextsim {
 
-static const std::map<int, std::string> keyMap
-    = { { Xios::ENABLED_KEY, "xios.enable" }, { Xios::START_TIME_KEY, "model.start" },
-          { Xios::TIME_STEP_KEY, "model.time_step" }, { Xios::PERIOD_KEY, "XiosOutput.period" },
-          { Xios::OUTPUT_FILENAME_KEY, "XiosOutput.filename" } };
+static const std::map<int, std::string> keyMap = { { Xios::ENABLED_KEY, "xios.enable" },
+    { Xios::START_TIME_KEY, "model.start" }, { Xios::TIME_STEP_KEY, "model.time_step" },
+    { Xios::OUTPUT_PERIOD_KEY, "XiosOutput.period" },
+    { Xios::OUTPUT_FILENAME_KEY, "XiosOutput.filename" },
+    { Xios::OUTPUT_FIELD_NAMES_KEY, "XiosOutput.field_names" },
+    { Xios::INPUT_PERIOD_KEY, "XiosInput.period" },
+    { Xios::INPUT_FILENAME_KEY, "XiosInput.filename" },
+    { Xios::INPUT_FIELD_NAMES_KEY, "XiosInput.field_names" } };
```

Comment:
A suggestion: parameterize the `"XiosInput"` and `"XiosOutput"` strings to give something like `XInputPfx + ".period"`.

Reply:
Good idea - done in db7618bcc19031967cdb6faff717756ae6f7ac75.